doc/manual/manual.tex

```diff
@@ -7795,6 +7795,43 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+
+\subsubsection{The Bunge et al.~mantle convection experiments}
+\label{sec:bunge_et_al_cookbook}
+
+\textit{This section was contributed by Cedric Thieulot and Bob Myhill.}
+
+Early mantle modelling studies of the 1970s, 1980s and 1990s were often concerned with simple set-ups (Cartesian geometries, incompressible fluid, free slip boundary conditions) and investigated the influence of the Rayleigh number, the heating mode or the temperature dependence of the viscosity on temperature, pressure and/or strain rate \cite{youn74,buss75,buss79,BBC89,BC93,vavy93,burb97}. In this cookbook, we use the `simple' material model to reproduce the set-up in \cite{burb96}, which reported that even modest increases in mantle viscosity with depth could have a marked effect on the style of mantle convection.
+
```

Comment:
Would you mind adding a sentence that the benchmark is ``\url{cookbooks/bunge_et_al_mantle_convection/bunge_et_al.prm}``?